server/controllers/request.js

```diff
@@ -0,0 +1,105 @@
+const Request = require("../models/Request");
+const asyncHandler = require("express-async-handler");
+
+// @route GET /requests
+// @desc load requests for logged in user
+// @access Private
+exports.loadRequests = asyncHandler(async (req, res, next) => {
+  const requests = await Request.find({ userId: req.user.id });
```

Comment:
```suggestion
  const requests = await Request.find({ sitterId: req.user.id });
```
If we wish to return request for a sitter, we'll need to filter by `sitterId`